sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/metrics/QueryMetricsTextWriter.java

```diff
@@ -21,101 +21,103 @@
 
     // QueryMetrics
     private static final String ActivityIds = "Activity Ids";
-    private static final String RetrievedDocumentCount = "Retrieved Document Count";
-    private static final String RetrievedDocumentSize = "Retrieved Document Size";
-    private static final String OutputDocumentCount = "Output Document Count";
-    private static final String OutputDocumentSize = "Output Document Size";
-    private static final String IndexUtilizationText = "Index Utilization";
-    private static final String TotalQueryExecutionTime = "Total Query Execution Time";
+    private static final String RETRIEVED_DOCUMENT_COUNT = "Retrieved Document Count";
+    private static final String RETRIEVED_DOCUMENT_SIZE = "Retrieved Document Size";
+    private static final String OUTPUT_DOCUMENT_COUNT = "Output Document Count";
+    private static final String OUTPUT_DOCUMENT_SIZE = "Output Document Size";
+    private static final String INDEX_UTILIZATION_TEXT = "Index Utilization";
+    private static final String TOTAL_QUERY_EXECUTION_TIME = "Total Query Execution Time";
 
     // QueryPreparationTimes
-    private static final String QueryPreparationTimes = "Query Preparation Times";
-    private static final String QueryCompileTime = "Query Compilation Time";
-    private static final String LogicalPlanBuildTime = "Logical Plan Build Time";
-    private static final String PhysicalPlanBuildTime = "Physical Plan Build Time";
-    private static final String QueryOptimizationTime = "Query Optimization Time";
+    private static final String QUERY_PREPARATION_TIMES = "Query Preparation Times";
+    private static final String QUERY_COMPILATION_TIME = "Query Compilation Time";
+    private static final String LOGICAL_PLAN_BUILD_TIME = "Logical Plan Build Time";
+    private static final String PHYSICAL_PLAN_BUILD_TIME = "Physical Plan Build Time";
+    private static final String QUERY_OPTIMIZATION_TIME = "Query Optimization Time";
 
     // QueryTimes
-    private static final String QueryEngineTimes = "Query Engine Times";
-    private static final String IndexLookupTime = "Index Lookup Time";
-    private static final String DocumentLoadTime = "Document Load Time";
-    private static final String DocumentWriteTime = "Document Write Time";
+    private static final String QUERY_ENGINE_TIMES = "Query Engine Times";
+    private static final String INDEX_LOOKUP_TIME = "Index Lookup Time";
+    private static final String DOCUMENT_LOAD_TIME = "Document Load Time";
+    private static final String DOCUMENT_WRITE_TIME = "Document Write Time";
 
     // RuntimeExecutionTimes
-    private static final String RuntimeExecutionTimes = "Runtime Execution Times";
+    private static final String RUNTIME_EXECUTION_TIMES = "Runtime Execution Times";
     private static final String TotalExecutionTime = "Query Engine Execution Time";
```

Comment:
TotalExecutionTime->TOTAL_EXECUTION_TIME?

Reply:
addressed in the lastest [commit](https://github.com/Azure/azure-sdk-for-java/pull/9111/commits/b972e8c088bb80842e7fa0633740f49128f1991b).